internal/web/assets/Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG BASE_IMAGE=ubuntu:20.04
+ARG KAIROS_INIT_VERSION=v0.2.3
+ARG KAIROS_INIT_IMAGE=quay.io/kairos/kairos-init:${KAIROS_INIT_VERSION}
+
+FROM ${KAIROS_INIT_IMAGE} AS kairos-init
+
+FROM ${BASE_IMAGE} AS base
+ARG VARIANT=core
+ARG MODEL=generic
+ARG TRUSTED_BOOT=false
+ARG KUBERNETES_PROVIDER=k3s
+ARG KUBERNETES_VERSION=latest
```

Comment:
The kairos one has already diverged: https://github.com/kairos-io/kairos/blob/34c05ee585cb9adbd402f6c084e5a20d40bf9199/images/Dockerfile#L12

(accepts `VERSION` arg)

I wonder what is the canonical location for the dockerfile now. Since kairos repo is also using Auroraboot, maybe kairos should also use this one?

Reply:
got rid of it for now